frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -136,7 +141,7 @@ export default class TransactionsApi extends ExplorerApi {
       const transactions = await this.call<
         (TransactionInfo & (StringActions | Actions))[]
```

Comment:
So, since we don't query actions here now, we should just expect `TransactionInfo[]` type here. And as a result, we can remove `StringActions` interface.